rpcs3/rpcs3qt/settings_dialog.ui

```diff
@@ -1725,6 +1725,13 @@
               </property>
              </widget>
             </item>
+            <item>
+			          <widget class="QCheckBox" name="enableTSX">
```

Comment:
why did you add three tabs to all of these already (with spaces) indented lines

Reply:
Don't know why this happened lol